### ISSUE TITLE
Spelling mistake in gendefer.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6418]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6418|
|Type|defect|
|Created|2013-04-07 23:42:14Z|

`core/doc/howto/gendefer.xhtml` has a spelling mistake: "possiblity".

Attachments:

* [6418.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/bd7/bd7847e688cbf56ffb9d0de7512cc88b90f6d122/1717279c88311b38369b5d4bf98f176c639009b7.patch) (699 bytes) - added by introom on 2013-04-08 12:43:29Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6418 6418
type__defect defect
reporter__itamar itamar
priority__lowest lowest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1365378134000000 1365378134000000
changetime__1365770775000000 1365770775000000
version__None None
owner__thijs thijs

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks, I'll apply it.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38045]) Apply 6418.patch: Correct a typo in the Generating Deferreds howto.

Author: introom
Reviewer: thijs
Fixes: [#6418](https://github.com/twisted/twisted/issues/6418)
